### PR TITLE
Template cross selling

### PR DESCRIPTION
**Pull request by rotorrest**
### Description

This PR contains the Coss Selling template. 

### Added
-ABC
- ABC
### Changed
- ABC
- ABC

### Fixed
- ABC
- ABC

### CODE REVIEW

File: templates/cross_selling/.env [hunk 1]
```diff
@@ -0,0 +1,3 @@
+SHIMOKU_TOKEN=4a6dfc4d-051f-41e2-9c3b-5fe0a6328007
+UNIVERSE_ID=11f75ee7-51bf-41e4-9f2f-ef7a9c109430
```
Comment (dgibert17) [inline templates/cross_selling/.env:2]: NOOOOOOOOOOOOOOO seguridad comprometidaaaaaaa

**Review by dgibert17 — CHANGES_REQUESTED** [review]
No mandes las keys locoooo

File: templates/cross_selling/.env (hunk 1 above)
Comment (rotorrest) [reply templates/cross_selling/.env]: Hola @dgibert17, elimine el file en este commit
https://github.com/shimoku-tech/shimoku-app-templates/pull/13/commits/a8d601c4dc36595e962987bc575c8687036db7b7

**Review by dgibert17 — APPROVED** [review]
estupendamente

**Pull request merged**